dbmigrator/utils.py

```diff
@@ -206,6 +210,18 @@ def run_migration(cursor, version, migration_name, migration):
     except AttributeError:
         # not a repeat migration
         pass
+
+    if not run_defers:
+        try:
+            if migration.up.dbmigrator_deferrable:
+                print('Marking migration {} {} as deferred'
+                      .format(version, migration_name))
+                mark_migration(cursor, version, 'deferred')
```

Comment:
The only advantage would be when you do `dbmigrator list`, you can see that the `is applied` is `deferred` instead of `False`.